mail-client/bower/files/makefile.patch

```diff
@@ -0,0 +1,20 @@
+--- a/src/Makefile	2018-11-10 21:09:38.939492000 +0100
++++ b/src/Makefile	2018-11-11 15:47:10.668689276 +0100
+@@ -1,14 +1,13 @@
+ MMC = mmc
+-PARALLEL =
```

Comment:
Why are you removing this? The way you pass it should override the empty default.